.gitignore

```diff
@@ -13,4 +13,5 @@
 ###> friendsofphp/php-cs-fixer ###
 /.php_cs
 /.php_cs.cache
+/.php-cs-fixer.cache
```

Comment:
I have a feeling that one of `/.php_cs.cache` and `/.php-cs-fixer.cache` can be removed.

Reply:
`/.php_cs.cache` can indeed be removed and has been removed.